src/tests/BUILD.gn

```diff
@@ -156,6 +156,17 @@ source_set("webnn_end2end_tests_sources") {
   configs += [ "${webnn_root}/src/common:dawn_internal" ]
   testonly = true
 
+  _models_folder_relative_path =
+      "../../node/third_party/webnn-polyfill/test-data/models/"
+
+  _data_path = rebase_path(_models_folder_relative_path, webnn_root)
+  _webnn_root_data_path =
+      get_path_info("${webnn_root}" + "${_data_path}", "dir")
+  _models_folder_absolute_path = rebase_path(_webnn_root_data_path)
```

Comment:
Maybe the end_2_end can't be run if all binaries are packaged to run in other directory?

Reply:
@fujunwei, we should be able to run the end2end tests from any directory (eg. under .\third_party), so long as they don't assume to load files using relative paths (ex. node\folder).

This is how `_models_folder_absolute_path ` gets set, eg. `C:\Users\bryan\src\webnn_native\node\models` or `C:\Users\bryan\src\<some project>\third_party\webnn_native\node\modes`, depending on webnn_root.

Reply:
We will public webNN-native with node.js to npm, users get the package and install in their local system that is not the webnn_root(C:\Users\bryan\src\), it will fail to run?

Reply:
@fujunwei, we can't run these tests with an unresolved install-time path (custom path vs DEPS). This change assumes DEPS path is used, like how BUILD.gn locates source-files. If the user wants to install their own custom `node` package, they (still) need to use the same path (install under `.\node`) or say, be given an option to override the DEPS path.

Reply:
@fujunwei PTAL. We cannot run end2end tests under `.\third_party` without this fix.

Reply:
Let's merge it first, we can add the relative path option to run model test when we have the requirement. Thanks.